core/tests/org.fusesource.ide.camel.model.service.core.tests.integration/src/main/java/org/fusesource/ide/camel/model/service/core/tests/integration/core/model/CamelBasicModelElementTestIT.java

```diff
@@ -0,0 +1,66 @@
+package org.fusesource.ide.camel.model.service.core.tests.integration.core.model;
+
+import static org.fusesource.ide.preferences.PreferencesConstants.EDITOR_USER_LABELS;
+import static org.fusesource.ide.preferences.PreferencesConstants.EDITOR_PREFER_ID_AS_LABEL;
+
+import org.fusesource.ide.camel.model.service.core.catalog.eips.Eip;
+import org.fusesource.ide.camel.model.service.core.model.AbstractCamelModelElement;
+import org.fusesource.ide.camel.model.service.core.model.CamelBasicModelElement;
+import org.fusesource.ide.preferences.PreferenceManager;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * 
+ * @author Andrej Podhradsky (apodhrad@redhat.com)
+ *
+ */
+public class CamelBasicModelElementTestIT {
+
+	@Before
+	@After
+	public void cleanUserLabelsPreference() {
+		PreferenceManager.getInstance().savePreference(EDITOR_USER_LABELS, "");
+		PreferenceManager.getInstance().savePreference(EDITOR_PREFER_ID_AS_LABEL, false);
+	}
+
+	@Test
+	public void testDisplayingDefaultText() {
+		Assert.assertEquals("Log coolName", simpleLog().getDisplayText());
```

Comment:
you can use static import for Assert.assertEquals (or even Assertions.assertThat if using AssertJ API)
See http://tools.jboss.org/blog/working-efficiciently-junit-eclipse-update.html how to configure automatic static import
(or Ctrl+Shift+M if I remember well when it was already written)